workflows/dags/author/author_create/author_create_rejected.py

```diff
@@ -28,24 +28,20 @@ def author_create_rejected_dag() -> None:
     2. set_author_create_workflow_status_to_completed: Sets the status of
         the author creation workflow to 'completed'.
     """
-    workflow_management_hook = WorkflowManagementHook()
+    workflow_management_hook = WorkflowManagementHook(AUTHORS)
 
     @task()
     def set_author_create_workflow_status_to_completed(**context: dict) -> None:
         status_name = "completed"
         workflow_management_hook.set_workflow_status(
-            status_name=status_name,
-            workflow_id=context["params"]["workflow_id"],
-            collection=AUTHORS,
+            status_name=status_name, workflow_id=context["params"]["workflow_id"]
```

Comment:
have you enabled pre-commit? 🤔

Reply:
yes. It was not working I fixed it and reran it. I believe it was it that made this change